pipelines/deploy-infrastructure.yml

```diff
@@ -1,12 +1,37 @@
 trigger:
-  - master
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - deployment/*
+      - pipelines/deploy-infrastructure.yml
 
 pr: none
 
 pool:
   vmImage: "ubuntu-latest"
 
+variables:
+  globalArguments: prodoh urlist dev westus2 eastus centralus
+  azureSubscription: PELITTLE TEAM - CSE DWR(d36d0808-a967-4f73-9fdc-32ea232fc81d)
```

Comment:
I would replace all reference to azureSubscription to referencing a variable group in AzDO.  In case we have to switch to a different subscription, we only have to change it once.

Reply:
Good point. Will do.